src/components/thumbnail/tab-panel-documents-section.sass

```diff
@@ -0,0 +1,121 @@
+@import ../vars
+
+$list-item-height: 660px
+$list-item-width: 880px
+$list-item-scale: 0.11
+
+.tab-panel-documents-section
+  height: 100%
+  .list
+    height: calc(100vh - (#{$header-height} + #{$workspace-content-margin} * 2 + #{$left-tab-height} * 2))
+    display: flex
+    flex-wrap: wrap
+    align-content: flex-start
+    overflow: auto
+    overflow-x: hidden
+    padding: $padding
+
+    .list-item
+      position: relative
+      min-height: 120px
+      width: 116px
+      border-radius: 5px
+      padding: $padding 0
+      display: flex
+      flex-direction: column
+      align-items: center
+      background-color: white
+      overflow: hidden
+      cursor: pointer
+
+      .scaled-list-item-container
+        border-radius: 5px
+        border: solid 1.5px $charcoal-light-1
+        width: $list-item-width * $list-item-scale
+        height: $list-item-height * $list-item-scale
+        background-color: white
+        padding: 1px
+
+        .scaled-list-item
+          transform: scale(.1)
+          transform-origin: 0 0
+          height: $list-item-height
+          width: $list-item-width
+
+        &.new-document-button
+          position: relative
+
+          .new-document-button-label
+            position: absolute
+            left: 2px
+            top: 18px
+            width: 100%
+            height: 100%
+            display: flex
+            flex-direction: column
+            align-items: center
+            color: #C9DAB7
```

Comment:
Is there a named constant for this color? There are a few other magic number colors in this file that could use constants as well.

Reply:
I believe these instances are still pieces from the old style spec (this is one of those pale green colors) that we haven't gotten to yet.  I think we should leave these in for now as it is an indication of a piece that still needs to be converted to the new UI spec (in this case it's the new doc button that hasn't been worked on yet).